discovery-provider/plugins/network-monitoring/src/metrics/queries.ts

```diff
@@ -286,3 +286,28 @@ export const getUsersWithEntireReplicaSetInSpidSetCount = async (run_id: number,
 
     return usersCount
 }
+
+export const getUsersWithEntireReplicaSetNotInSpidSetCount = async (run_id: number, spidSet: number[]): Promise<number> => {
```

Comment:
idr if you checked this earlier but how (ballpark) long do these queries take to run against prod dataset?

(just respond in slack)